engines/mohawk/detection_tables.h

```diff
@@ -3042,6 +3058,21 @@ static const MohawkGameDescription gameDescriptions[] = {
 		0,
 	},
 
+	{
+		{
+			"create",
```

Comment:
Can you add a comment that this is for the demo version? Might be helpful for the future.